timescaledb/docker-dependencies/timescaledb-toolkit-pg17

```diff
@@ -0,0 +1,60 @@
+FROM timescale/timescaledb:2.17.2-pg17 as timescale-pg17
+
+FROM timescale-pg17 AS timescaledb-toolkit
+ARG VERSION=1.19.0
+ARG TARGETARCH
+RUN --mount=type=cache,id=apk-${TARGETARCH}-${TARGETVARIANT},sharing=locked,target=/var/cache/apk \
+    --mount=type=cache,id=cargo-${TARGETARCH}-${TARGETVARIANT},sharing=locked,target=/root/.cargo \
+    set -ex \
+    && apk update \
+    && if [ "$TARGETARCH" == "amd64" ] || [ "$TARGETARCH" == "arm64" ]; then \
+            apk add \
+                        ca-certificates \
+                        git \
+                        openssl \
+                        openssl-dev \
+                        tar \
+                        wget \
+                        curl \
+            && apk add \
+                        coreutils \
+                        gcc \
+                        make \
+                        cmake \
+                        build-base \
+            && apk add --repository http://dl-cdn.alpinelinux.org/alpine/v3.20/main/ \
```

Comment:
TODO: test using the rust version of Alpine 3.21, maybe it's fixed there?

Reply:
Haha uugh.,.

You don't know how pleased I am that at least somebody else is also fighting this battle! It was pretty darn lonely here in the trenches at times 🤣

Reply:
Here you go! The increased stackl size actually worked! 🎉🤣
![image](https://github.com/user-attachments/assets/5120ce0e-1cd5-4309-a1ad-4bbeeb8daea9)

Reply:
It took me freaking 4 hours to compile all versions of timescaledb-toolkit for the supported architectures..